src/config/searchWorker.ts

```diff
@@ -16,29 +16,86 @@
 
 /* eslint-disable no-restricted-globals */
 export default () => {
-    function debounceSearch(callback: (...args: any[]) => void) {
-        let timeout
-        return (...args: any[]): void => {
+    /**
+     * Compares two values for sorting.
+     * @param  a - The first value to compare.
+     * @param  b - The second value to compare.
+     * @returns  The comparison result (-1 if a < b, 0 if a = b, 1 if a > b).
+     */
+    const comparator = (a: string | number | object, b: string | number | object) => {
+        let result = 0
+        if (a < b) {
+            result = -1
+        } else if (a > b) {
+            result = 1
+        }
+        return result
+    }
```

Comment:
- How would this be consistent for string, number and object?
- Please check stringComparatorBySortOrder and numberComparatorBySortOrder if applicable